src/main/resources/mail.properties

```diff
@@ -0,0 +1,7 @@
+#mail
```

Comment:
In practice no need to create separate property file, please move this to application.properties